Lib/test/libregrtest/main.py

```diff
@@ -533,6 +534,13 @@ def main(self, tests=None, **kwargs):
     def _main(self, tests, kwargs):
         self.ns = self.parse_args(kwargs)
 
+        self.getloadavg = None
+        if hasattr(os, 'getloadavg'):
+            self.getloadavg = lambda: os.getloadavg()[0]
```

Comment:
I dislike lambda. Would you mind to define a method here using "def" ?

Reply:
I prefer lambda for this sort of short stuff but done